product-service/jest.config.ts

```diff
@@ -0,0 +1,198 @@
+/*
+ * For a detailed explanation regarding each configuration property and type check, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+import { pathsToModuleNameMapper } from 'ts-jest';
+import { compilerOptions } from "./tsconfig.paths.json";
+
+export default {
+  // All imported modules in your tests should be mocked automatically
+  // automock: false,
+
+  // Stop running tests after `n` failures
+  // bail: 0,
+
+  // The directory where Jest should store its cached dependency information
+  // cacheDirectory: "C:\\Users\\Viktar_Svintsitski\\AppData\\Local\\Temp\\jest",
+
+  // Automatically clear mock calls, instances, contexts and results before every test
+  clearMocks: true,
+
+  // Indicates whether the coverage information should be collected while executing the test
```

Comment:
Why do you need a million comments? =) Usually we post some comments in case we have some very complicated logic or using some "hacks".

Reply:
That was auto generated. I removed commented out code.